src/compiler/checker.ts

```diff
@@ -14301,11 +14301,11 @@ namespace ts {
                         candidateName === "set") {
                         continue;
                     }
-                    const distance = levenshtein(name, candidateName);
+                    const distance = dameraulevenshtein(name, candidateName);
                     if (distance > worstDistance) {
                         continue;
                     }
-                    if (distance < 3) {
+                    if (distance <= MIN_LEV_DIST) {

```

Comment:
for efficiency's sake, we should keep the early exit at 2, to include insertions (and double insertions).